common/djangoapps/student/models.py

```diff
@@ -2735,6 +2735,10 @@ def activate(self):
 class AccountRecovery(models.Model):
     """
     Model for storing information for user's account recovery in case of access loss.
+
+    .. pii: the field named secondary_email contains pii
```

Comment:
It would be nice for this to note that it is retired in the `DeactivateLogoutView`, but not a blocker.

Reply:
thanks - will do